components/schema-migrator/main.go

```diff
@@ -98,9 +101,30 @@ func invokeMigration() error {
 		return fmt.Errorf("# COULD NOT ESTABLISH CONNECTION TO DATABASE WITH CONNECTION STRING: %w", err)
 	}
 
+	log.Println("# STARTING COPY MIGRATION FILES #")
+	migrationTempPath := fmt.Sprintf("tmp-migrations-%s-*", os.Getenv("MIGRATION_PATH"))
+
+	tmpDir, err := os.MkdirTemp("/migrate", migrationTempPath)
+	if err != nil {
+		return fmt.Errorf("# COULD NOT CREATE TEMPORARY FILE FOR MIGRATION: %w", err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	migrationNewPath := fmt.Sprintf("new-migrations/%s", os.Getenv("MIGRATION_PATH"))
+	err = copyDir(migrationNewPath, tmpDir)
+	if err != nil {
+		log.Printf("# COULD NOT COPY NEW MIGRATION FILES: %s\n", err)
```

Comment:
I am afraid that we might hide some errors in such ways. Please consider filtering error that is expected logging the rest of them with `fmt.Error` or errors.